.github/workflows/publish.yml

```diff
@@ -0,0 +1,143 @@
+name: Publish Version
+
+on:
+  workflow_call:
+    inputs:
+      simulate:
+        required: false
+        type: boolean
+        default: true
```

Comment:
default=false tyvm

Reply:
I set the flag explicitly when called in `publish-on-release` but sure.